717 - Add metadata only download readme (#744)
AlexsLemonade/scpca-portal: api/scpca_portal/templates/readme/metadata_only.md

> {% load custom_filters %}

Reviewer: ```suggestion
```

Generated on: {{ date }}

# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas

The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).

## Contents

> {% if entire_portal %}

Reviewer: ```suggestion
{% if len(included_projects) > 1 %}
```

This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
{% else %}
This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
{% endif %}

> 

Reviewer 2: ```suggestion
The metadata included in this download contains sample, library, and project-related metadata (e.g., age, sex, diagnosis, sequencing unit, etc.) along with any relevant processing metadata (e.g., software versions, filtering methods used, etc.). 
```
Probably useful to expand on the contents of what they can expect in the metadata slightly although the full description is in the docs link.

Author: Great! Added.

> This download includes associated metadata for samples from project {{ included_projects|markdown_links }} in the ScPCA portal.

Reviewer: ```suggestion
{% for proejct_accession, project_url in included_projects %}
This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
{% endfor %}
```

Let's just keep this simple. For now we only anticipate two use cases:
- All projects
- One project

We can rewrite it to make sense for an arbitrary amount of projects later.

## CHANGELOG

A summary of changes impacting downloads from the ScPCA Portal is available in [the CHANGELOG section of our documentation](https://scpca.readthedocs.io/en/stable/CHANGELOG.html).

## Contact

If you identify issues with this download, please [file an issue on GitHub.](https://github.com/AlexsLemonade/scpca-portal/issues/new) If you would prefer to report issues via e-mail, please contact us at [scpca@ccdatalab.org](mailto:scpca@ccdatalab.org).

## Citation

If you use these data in your research, you must cite:

- The data submitter using language provided as part of the project abstract (as applicable), the publication listed for the project (as applicable), or both.
- The ScPCA Portal using the language below.

For more information, please see [the How to Cite section of our documentation](https://scpca.readthedocs.io/en/stable/citation.html).

### Citing this project

{% for project_accession, project_url in included_projects %}
To cite data from {{ project_accession }}, please see the project abstract and publication information at [{{ project_accession }} page.]({{ project_url }})

> {% endfor %}

Author: @davidsmejia
If there are multiple projects, do we want multiple line 40s, or do we want to handle this in a similar way to line 13?

Reviewer: nope this looks correct

### Citing the ScPCA Portal

When citing the ScPCA Portal, please cite the following preprint:

> Hawkins A. G., J. A. Shapiro, S. J. Spielman, D. S. Mejia, D. V. Prasad, et al., 2024 The Single-cell Pediatric Cancer Atlas: Data portal and open-source tools for single-cell transcriptomics of pediatric tumors. _bioRxiv._ https://doi.org/10.1101/2024.04.19.590243

Reviewer 3: Just noting inconsistent initial/last name order here

Reviewer 2: This is correct. We use [genetics citation style](https://paperpile.com/s/genetics-citation-style/).

Reviewer 3: wow, I never knew that! cool

Generated on: {{ date }}

# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas

The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).

## Contents

> {% if project_accession %}

Reviewer: we shouldn't pass `project_accession` here we should expect:

`included_projects` - a list of tuples where the first item is the accession and the second is the url

We should instead check that the `included_projects` is greater than 1 and display the "else" content here
If it is only 1 just iterate over the tuples.

> {% if project_accession %}
> This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
> {% else %}
> This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
> {% endif %}

Reviewer: ```suggestion
{% if entire_portal %}
This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
{% else %}
{% for project_accession, project_url in included_projects %}
This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
{% endfor %}
{% endif %}
```

We can add a new attribute that says it is for everything in the portal.
